flytekit/experimental/eager_function.py

```diff
@@ -396,6 +398,8 @@ def eager(
     :param local_entrypoint: If True, the eager workflow will can be executed locally but use the provided
         :py:func:`~flytekit.remote.FlyteRemote` object to create task/workflow executions. This is useful for local
         testing against a remote Flyte cluster.
+    :param client_secret_env_var: if specified, binds the client secret to the specified environment variable for
```

Comment:
okay with this for now, but I think we should see if we can't move this to some higher level in the future.  the secret here we're talking about is the secret for hitting admin itself right?